openlibrary/templates/books/edit.html

```diff
@@ -34,13 +34,15 @@
         else:
             title = _("Editing...")
             note = ""
-    <h2 class="editFormTitle">$:title</h2>
     $if ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/super-librarians')):
+        $:macros.databarEdit(work)
```

Comment:
```suggestion
    $:macros.databarEdit(work)
    $if ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/super-librarians')):
```

Actually, we want to show the cancel button to users even if they're not an admin or librarian.
Sorry for the confusion.

Reply:
@Realmbird can you make this change?